### DISCUSSION TITLE
Formatting async with none-ls

### DISCUSSION THREAD

**Discussion by rpbaptist** [LazyVim/LazyVim#4651] [Q&A]
Currently in a large Elixir project, formatting async with none-ls is pretty much unbearable with auto-save and format on save turned on.

There is a [code example on none-ls for formatting async](https://github.com/nvimtools/none-ls.nvim/wiki/Formatting-on-save), but I'm a bit lost on how to insert that into the LazyVim config. Can anyone provide an example?

**Comment by dpetka2001** [comment]
I don't use `none-ls`, but from what I can tell the simplest solution for you would be to create a new keymap that calls `vim.lsp.buf.format({ async = false })` and use that instead.

**Comment by rpbaptist** [comment]
I missed the transition from `none-ls` to `conform` and I was using `none-ls` as an extra. I disabled that and it's all good now.
